extension/src/installPrompt.ts

```diff
@@ -40,19 +40,25 @@ export class InstallPrompt {
     }
 
     if (selection === this.installMessage) {
+      let themeExtension = vscode.extensions.getExtension(this.theme.extensionName);
+      if(themeExtension == undefined){
+        await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification },
+          (progress) => {
+            progress.report({ message: `Installing ${this.theme.name} theme...` });
+  
+            return new Promise((resolve) => {
+              vscode.extensions.onDidChange((e) => resolve(null));
+              vscode.commands.executeCommand("workbench.extensions.installExtension", this.theme.extensionName);
+            });
+          },
+        );
+        themeExtension = vscode.extensions.getExtension(this.theme.extensionName);  
+      } else {
```

Comment:
Let's remove this informational message. I believe the next thing we do is set the theme. That's enough notification.

Reply:
Updated